### DISCUSSION TITLE
GirlScript | AMA Session

### DISCUSSION THREAD

**Discussion by vinitshahdeo** [vinitshahdeo/vinitshahdeo#2] [General]
[![image](https://user-images.githubusercontent.com/20594326/105574776-d7aa5100-5d8c-11eb-90e8-a835deb7a261.png)](https://github.com/vinitshahdeo/)

# Hello :wave:

### I'm honoured to be invited for an exclusive Q&A session, organized by [GirlScript Ghaziabad](https://twitter.com/gscriptGZB)/NCR. 

> Learn more about GirlScript [here](https://www.girlscript.tech/home). **Read my open-source journey [here](https://dev.to/vinitshahdeo/my-journey-from-0-to-1k-followers-on-github-1n6h)**.

```javascript

  ___      _     ___  ___                  
 / _ \    | |    |  \/  |                  
/ /_\ \___| | __ | .  . | ___              
|  _  / __| |/ / | |\/| |/ _ \             
| | | \__ \   <  | |  | |  __/             
\_| |_/___/_|\_\ \_|  |_/\___|             
                                           
                                           
  ___              _   _     _             
 / _ \            | | | |   (_)            
/ /_\ \_ __  _   _| |_| |__  _ _ __   __ _ 
|  _  | '_ \| | | | __| '_ \| | '_ \ / _` |
| | | | | | | |_| | |_| | | | | | | | (_| |
\_| |_/_| |_|\__, |\__|_| |_|_|_| |_|\__, |
              __/ |                   __/ |
             |___/                   |___/ 

```

## Feel free to shoot your doubts here.

`<Keep Coding! />`
[![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://github.com/vinitshahdeo)

🤗 

[![Twitter Follow](https://img.shields.io/twitter/follow/Vinit_Shahdeo?style=social)](https://twitter.com/Vinit_Shahdeo)

**Comment by PatrickZablocki** [comment]
I like your profile, looks very good
